### PR TITLE
Bug 1615660 Added note box on networkpolicy not allowing egress

### PR DESCRIPTION
**Pull request by bfallonf**
For: https://bugzilla.redhat.com/show_bug.cgi?id=1615660

Will verify in the BZ.

### CODE REVIEW

File: admin_guide/managing_networking.adoc [hunk 1]
```diff
@@ -1289,9 +1289,14 @@ the case with unicast.
...
-The *ovs-subnet* and *ovs-multitenant* plug-ins have their own legacy models of network
-isolation, and do not support Kubernetes `NetworkPolicy`. However, `NetworkPolicy` support
-is available by using the *ovs-networkpolicy* plug-in.
+The *ovs-subnet* and *ovs-multitenant* plug-ins have their own legacy models of
+network isolation, and do not support Kubernetes `NetworkPolicy`. However,
+`NetworkPolicy` support is available by using the *ovs-networkpolicy* plug-in.
+
+[NOTE]
+====
+Only the `v1` NetworkPolicy features are available in {product-title}. This means that egress policy types, IPBlock, and combining `podSelector` and `namespaceSelector` opjects are not available in {product-title}.
```
[the remark this answers is not in the document]
Comment (bfallonf) [reply admin_guide/managing_networking.adoc]: Sure. Thanks @nekop . I've removed the word so it just says the features are not available.

**Comment by bfallonf** [comment]
Okayed by reporter. @openshift/team-documentation PTAL

File: admin_guide/managing_networking.adoc [hunk 2]
```diff
@@ -1289,9 +1289,16 @@ the case with unicast.
 [[admin-guide-networking-networkpolicy]]
 == Enabling NetworkPolicy
 
-The *ovs-subnet* and *ovs-multitenant* plug-ins have their own legacy models of network
-isolation, and do not support Kubernetes `NetworkPolicy`. However, `NetworkPolicy` support
-is available by using the *ovs-networkpolicy* plug-in.
+The *ovs-subnet* and *ovs-multitenant* plug-ins have their own legacy models of
+network isolation, and do not support Kubernetes `NetworkPolicy`. However,
```
Comment (kalexand-rh) [inline admin_guide/managing_networking.adoc]: s/isolation,/isolation

**Review by kalexand-rh — COMMENTED** [review]
One pick. LGTM

**Pull request merged**